webapp/components/Notification/Notification.vue

```diff
@@ -70,14 +56,36 @@ export default {
 </script>
 
 <style lang="scss">
-.notification.read {
-  opacity: $opacity-soft;
-}
-.notifications-card {
-  min-width: 500px;
-}
-.comment-notification-header {
-  font-weight: 700;
-  margin-right: 0.1rem;
+.notification {
+  margin-bottom: $space-base;
+
+  &:first-of-type {
+    margin-top: $space-x-small;
+  }
+
+  &.--read {
+    opacity: $opacity-disabled;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-12T13:15:57Z" title="Wednesday, February 12th 2020, 2:15:57 pm +01:00">Feb 12, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -70,14 +56,36 @@ export default {
 </script>
 
 <style lang="scss">
-.notification.read {
-  opacity: $opacity-soft;
-}
-.notifications-card {
-  min-width: 500px;
-}
-.comment-notification-header {
-  font-weight: 700;
-  margin-right: 0.1rem;
+.notification {
+  margin-bottom: $space-base;
+
+  &:first-of-type {
+    margin-top: $space-x-small;
+  }
+
+  &.--read {
+    opacity: 0.5;
```


```suggestion
    opacity: $opacity-disabled;
```
avoid magic numbers?